future-apps/payroll/app/src/screens/components/SalaryTable.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import styled from 'styled-components'
+import Table from '../../components/Table'
+import { salaryType } from '../../types'
+import { formatDate, formatTokenAmount } from '../../utils/formatting'
+
+import { Button, theme } from '@aragon/ui'
+
+const initializeColumns = (data, formatExchangeRate) => {
+  return [
+    {
+      name: 'date',
+      title: 'Date',
+      value: data => data.date,
+      formatter: formatDate
+    },
+    {
+      name: 'status',
+      title: 'Status',
+      value: data => data.status,
+    },
+    {
+      name: 'transaction-address',
+      title: 'Transaction Address',
+      value: data => data.transactionAddress,
+      defaultValue: 'Active',
+      render: (formattedValue, rawValue, item) => (
+        <TransactionAddress>
```

Comment:
We could use aragon-ui `IdentityBadge` component here, it's the one being used in Finance app

Reply:
We're not using IdentityBadge but we're planning on create a custom component using SafeLink. The approach is pushed